### ISSUE TITLE
Add last block time to the status

### ISSUE THREAD

**Issue by chevdor** [ethcore/parity#3170]
Especially during long sync, that allows users to see how far the sync is in a unit that is more human friendly: time.

![screen_shot_2016-11-04_at_11_16_01](https://cloud.githubusercontent.com/assets/738724/20002250/478a4fc4-a280-11e6-8400-ba012c3fafb2.png)

**Comment by jacogr** [comment]
Related https://github.com/ethcore/parity/issues/3114

**Comment by chevdor** [comment]
@jacogr I suggest reopening this issue.

Just tested and saw 2 things that look strange. I was on ropsten, with a few days to sync.
I started the sync and the block numbers started increasing (all good).
However, from the start, the timestamp showed the datetime or NOW (apparently). Is the data that is displayed really the date/time in the currently latest block?

Second little details I saw: during the sync, I saw "Today 17:42" pretty much all the time. Then it switched to "Tomorrow 17:42" (!?!!?) then came back to "Today 17:42" and the sync finished. The sync was too fast (Did I just write that?? :)) and I could catch a screenshot.

To confirm, I ran:
   `target/release/parity --testnet --datadir /tmp/junk ui`

And could see this:
![screen shot 2016-12-11 at 17 47 14](https://cloud.githubusercontent.com/assets/738724/21081559/fffe6048-bfc9-11e6-8db6-b76a06ced3d2.png)

I would have expected to see November 20th, 2016: https://testnet.etherscan.io/block/3175
